### DISCUSSION TITLE
[New Feature] BurstDelayX on AircraftTypes

### DISCUSSION THREAD

**Discussion by ghost** [Phobos-developers/Phobos#1001] [Ideas and feedback]
### Description:
I was working on an AircraftType today with Ares and realized that you can't use BurstDelayX on AircraftTypes. It would be nice if Phobos could make that possible.

**Comment by ComradeJarif** [comment]
Hello! I was the one who made this post more than a year ago. Was wondering if there's any updates about this.

**Comment by Starkku** [comment]
It is not a very simple thing to add in due to the way aircraft weapon firing works, and it would require reworking / reimplementing that to an extent. Perhaps at some point someone (probably me) will take a look at it but no promises.
